book/plugins/general-abilities/video-content-download/ios/video-content-download-ios.md

```diff
@@ -0,0 +1,156 @@
+## Video Content Downloads
+### Availible from SDK: 11.2.0
+
+1. <a href="#description">Description</a>
+2. <a href="#general">General</a>
+3. <a href="#implementation">Download button protocol implementation</a>
+4. <a href="#download-button-states-images3
+">Download button states images</a>
+* * *
+
+<a name="description" />
+
+##### Description
+`Video Content download` ability uses the provided atom entry video url to download and save the content locally for offline playback.
```

Comment:
Please add also a more specific designation of the guide.
`Given a screen plugin with Atom Item or items, the following guide goes through how to implement and customize a download button as a part of a Screen, component or cell.`

Reply:
designation - I mean scope ...